gpt_playground/settings.py

```diff
@@ -555,3 +555,5 @@
 ]
 SLACK_BOT_NAME = env("SLACK_BOT_NAME", default="@ocs")
 SLACK_ENABLED = SLACK_CLIENT_ID and SLACK_CLIENT_SECRET and SLACK_SIGNING_SECRET
+
```

Comment:
The commit message is a bit misleading. I forgot to remove `CRYPTOGRAPHY_KEY` from it when I removed it from this file. The reason I'm putting this back is because I get invalid token issues from the cryptography library. I think since we don't load this variable in the dev environment, it causes issues. After loading it, it seems to be fixed